samples/k8s-offer-azure-vote-custom-meters/createUIDefinition.json

```diff
@@ -215,7 +215,7 @@
 						"defaultValue": "Cats",
 						"constraints": {
 							"required": true,
-							"regex": "^[a-z0-9A-Z]{1,30}$",
+							"regex": "^[a-z0-9A-Z ]{1,30}$",
```

Comment:
Not related to your PR. I just noticed README is missing in k8s-offer-azure-vote and k8s-offer-azure-vote-custom-meters application. Can you please add a README in both these folders?